src/FilenameParsing/FileIDHelperResolutionStrategy.php

```diff
@@ -40,6 +40,13 @@ class FileIDHelperResolutionStrategy implements FileResolutionStrategy
      */
     private $versionedStage = Versioned::DRAFT;
 
+    /**
+     * todo: needs documenting
```

Comment:
Document

Reply:
I don't think it's worth documenting methods implemented from an interface. The doc block would be identical to the interface's docblock.